lib/collections.go

```diff
@@ -948,6 +1020,44 @@ func zipLists(arg0, arg1 ref.Val) ref.Val {
 	return types.NewRefValMap(types.DefaultTypeAdapter, m)
 }
 
+func mapKeys(val ref.Val) ref.Val {
+	mapK, ok := val.(traits.Mapper)
+	if !ok {
+		return types.ValOrErr(mapK, "no such overload")
+	}
+	n, _ := mapK.Size().(types.Int)
```

Comment:
```suggestion
	n := mapK.Size().(types.Int)
```
If this is fails, we want to know about it.

Reply:
Returned new error when fails.